src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/PrimarySite/index.js

```diff
@@ -14,7 +14,7 @@ import { translations } from '../../../../../utils/locales/translations';
 
 const StepPrimarySetup = () => {
 
-	const { setDrawerActiveView, setIsSidebarOpened, setIsDrawerSuppressed, setIsHeaderNavigationEnabled } = useDispatch(
+	const { setDrawerActiveView, setIsDrawerSuppressed, setIsHeaderNavigationEnabled, setSidebarActiveView } = useDispatch(
```

Comment:
Line 40 has another useDispatch, I knw this existed earlier but can you put it here?